Resources/Private/Translations/en/Main.xlf

```diff
@@ -326,6 +326,14 @@
                     <source>Discarded {0} changes.</source>
                 </trans-unit>
             </group>
+            <trans-unit id="switchedBaseWorkspace" xml:space="preserve">
+                <source>Switched base workspace to "{workspace}".</source>
```

Comment:
Didn't check and I assume you _did_, but just to make sure: `{workspace}` as placeholder works here? (I only knew that we support numeric placeholders)

Reply:
Tried it out again to confirm and it looks good: 
![Bildschirmfoto 2024-06-14 um 15 26 38](https://github.com/neos/neos-ui/assets/91674611/eb5c9c8b-2f33-43c4-868a-7a19760992d9)